controller/impl_test.go

```diff
@@ -0,0 +1,31 @@
+package controller
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/aserto-dev/go-aserto"
+	api "github.com/aserto-dev/go-grpc/aserto/api/v2"
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNoConnectionOptionsController(t *testing.T) {
```

Comment:
What exactly is being tested here? I think some refactoring is needed in order to actually exercise the different code paths in the command loop.